### ISSUE TITLE
GNOME, tasmota

### ISSUE THREAD

**Issue by uap4** [toxic-repos/toxic-repos#135]
В новости про новый выпуск окружения рабочего стола GNOME https://release.gnome.org/45/#new-image-viewer во втором видео лозунги и флаги. Прямая ссылка https://release.gnome.org/45/loupe.webm
Обнаружено это как минимум при обсуждении новости на linux.org.ru

Ещё не нашёл в списке про tasmota. Разраб же там сразу русский язык заменил на английский
https://github.com/arendst/Tasmota/commits/a67898e8b344403be490cc5cfc681b54ff420d2a/tasmota/language/ru_RU.h
, а потом добавил саботажный модуль 
https://github.com/arendst/Tasmota/commits/b4f99bb74704e4a5f85b7ba9e03b126bf1c43320/tasmota/xdrv_96_blacklist.ino
Так-то модуль только лозунг выдавал, вроде, а более вредоносная часть закоментирована, но я думаю, что такая явная угроза саботажа(бомба заложена) должна быть запомнена как саботаж. Ну и вокруг этого тоже агитация шла, не только в коде.

_Originally posted by @uap4 in https://github.com/toxic-repos/discussions/discussions/1_

**Comment by JayDi85** [comment]
Сейчас никаких следов вредоносного кода у tasmota не находится.

**Comment by uap4** [comment]
> Сейчас никаких следов вредоносного кода у tasmota не находится.

Да, по ссылкам видно, что язык возвращён, а модуль отключен со словами "Nice excercise in stiring the crowd." и удалён. В описании при внесении в репозиторий это стоит отметить. Указать нужно лозунги, сломанную сборку, враждебные действия. До вредоносного ПО не дотягивает.

**Comment by sergiomarotco** [comment]
#140
